TwilioVerify/TwilioVerify/Sources/Domain/Challenge/PushChallengeProcessor.swift

```diff
@@ -37,53 +37,70 @@ class PushChallengeProcessor {
 
 extension PushChallengeProcessor: PushChallengeProcessorProtocol {
   func getChallenge(withSid sid: String, withFactor factor: PushFactor, success: @escaping ChallengeSuccessBlock, failure: @escaping TwilioVerifyErrorBlock) {
+    Logger.shared.log(withLevel: .info, message: "Getting challenge \(sid) with factor \(factor.sid)")
     challengeProvider.get(withSid: sid, withFactor: factor, success: success, failure: { error in
       failure(TwilioVerifyError.inputError(error: error as NSError))
     })
   }
   
   func updateChallenge(withSid sid: String, withFactor factor: PushFactor, status: ChallengeStatus, success: @escaping EmptySuccessBlock, failure: @escaping TwilioVerifyErrorBlock) {
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/f45e089425972bef076bacaa943e53d5cf8f3bce/TwilioVerify/TwilioVerify/Sources/Domain/Challenge/PushChallengeProcessor.swift#L46"></span>Function body should span 40 lines or less excluding comments and whitespace: currently spans 51 lines<br />
<code>function_body_length</code> <code>PushChallengeProcessor.swift:46</code></td>
    </tr>
  </tbody>
</table>